var/spack/repos/builtin/packages/py-pyfaidx/package.py

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyPyfaidx(PythonPackage):
+    """pyfaidx: efficient pythonic random access to fasta subsequences"""
+
+    homepage = "https://pypi.python.org/pypi/pyfaidx"
+    url      = "https://pypi.python.org/packages/e2/ce/311179c6c49ad29943a8de5e78705ef88eb8ba7620b78431f84a92ef6902/pyfaidx-0.5.3.1.tar.gz"
```

Comment:
You can replace this URL with https://pypi.io/packages/source/p/pyfaidx/pyfaidx-0.5.3.1.tar.gz and it will work for all versions.